### ISSUE TITLE
Font-size Property Restriction: No Support for 'pt' Units

### ISSUE THREAD

**Issue by Moath-Zaghdad** [SVG-Edit/svgedit#949]
**Describe the bug**

On version [7.2.0](https://unpkg.com/svgedit@7.2.0/dist/editor/index.html) you can create a new font element and set the font to 10pt or 10px without any issue, however on on [7.3.3](https://unpkg.com/svgedit@7.3.3/dist/editor/index.html) the font-size attribute gets converted to float, note this line
https://github.com/SVG-Edit/svgedit/blob/2f9f6e2f717819ef92ac1a9b049005bd2ff4bdbb/packages/svgcanvas/core/undo.js#L192
which use to be
https://github.com/SVG-Edit/svgedit/blob/f47759867af79031e5465d660cf7f90314ac4a3f/packages/svgcanvas/undo.js#L192

**Comment by jfhenon** [comment]
Good point. We need to review the function  changeSelectedAttributeNoUndoMethod

**Comment by jfhenon** [comment]
Fixed and released in v7.4.2.
